.pre-commit-config.yaml

```
@@ -9,34 +9,46 @@ ci:
 
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
+    rev: v2.31.0
     hooks:
       - id: pyupgrade
-        args: ["--py37-plus"]
+        args: ["--py39-plus"]
 
   - repo: https://github.com/adamchainz/django-upgrade
-    rev: '1.15.0'
+    rev: '1.4.0'
```

Keep an eye out for these, and if the version is lower on one repo, update it so that we are using the later versions everywhere. So what I'm seeing here makes it look like the pre-commit file in django-cms may need it's version upgrading.

```suggestion
    rev: '1.15.0'
```